main.js

```diff
@@ -1090,6 +1090,22 @@ ipc.on('set-media-permissions', (event, value) => {
   }
 });
 
+// Loki - Auto updating
+ipc.on('get-auto-update-enabled', event => {
+  const configValue = userConfig.get('autoUpdate');
+  // eslint-disable-next-line no-param-reassign
+  event.returnValue = typeof configValue !== 'boolean' ? true : configValue;
+});
+
+ipc.on('set-auto-update-enabled', (event, value) => {
+  userConfig.set('autoUpdate', !!value);
+
+  // Stop updater if user turned it off
+  if (!value) {
+    updater.stop();
+  }
```

Comment:
and if you turn it back on without restarting, should we start it up here too?

Reply:
Good idea.